js/service.js

```diff
@@ -178,6 +178,7 @@ function updateStoredData(requestType = `POST`, endpointRoute, data = {}) {
 
         request.setRequestHeader(`X-CSRFToken`, data.csrfmiddlewaretoken);
         request.setRequestHeader(`Content-Type`, `application/json`);
+        //   xhr.setRequestHeader(`X-Requested-With`, `XMLHttpRequest`);
```

Comment:
To code reviewer:
I saw this on Foundation site but I'm not sure if we need this for Pulse? Thoughts?

Reply:
not sure tbh, this header is generally used for CSRF protection with CORS. If this request is working without this header we can leave it as it is.